CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+Contributions are welcome. For all non-trivial changes, please create an issue first to help us track what is being worked on. PR titles should follow the [angular style](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits) for compatibility with our automated release tool.
```

Comment:
Can you elaborate on this part more. It needs to be clear to the reader of this that:

- what the choices are
- how each choice iterates the version number
- how this automatically updates the CHANGELOG

Reply:
✅